lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -46,7 +46,9 @@
                         gettext('View Teams in the %(topic_name)s Topic'),
                         { topic_name: this.model.get('name') }, true
                     ));
-                    return '<span class="sr">' + screenReaderText + '</span><i class="icon fa fa-arrow-right" aria-hidden="true"></i>';
+                    return '<span class="sr">' +
```

Comment:
This might be the cause of the quality error. Let's work on correcting this together on Tuesday.